core/validation/failures.py

```diff
@@ -364,8 +369,8 @@ class InvalidOutcomeProjectFailure(ValidationFailure):
     def to_message(self) -> tuple[str, str, str, str]:
         sheet = "Outcomes"
         section = self.section
-        if self.section == "Footfall Indicator":
-            cell_index = "Cell unavailable for this section"
+        if section == "Footfall Indicator":
+            cell_index = "B" + str(self.row_indexes[0])  # different col for footfall indicator
```

Comment:
It's annoying that these cases don't fit into our `construct_cell_index` pattern.. I wonder if there is a way of managing it.. Will think again tomorrow on this

Reply:
Passing some kind of flag for footfall outcomes? I believe it's the only case where a column in the spreadsheet is different to a column in our tables

Reply:
A flag feels really coupling. This is a more dynamic way of doing it but could lead to the mappings becoming less interpretable.. what do you think?

see `additional_lookups`
```
def construct_cell_index(table: str, column: str, row_indexes: list[int], additional_lookups: tuple = ()) -> str:
    """Constructs the original Excel cell index of an error from the column and rows it occurred in.

    :param table: the internal table name where the error occurred
    :param column: the internal column name where the error occurred
    :param row_indexes: list of row indexes where the error occurred
    :param additional_lookups: additional lookups
    :return: indexes tuple of constructed letter and number indexes
    """
    column_letter = TABLE_AND_COLUMN_TO_ORIGINAL_COLUMN_LETTER[table][column]
    for lookup in additional_lookups:
        column_letter = column_letter[lookup]
    # remove duplicate row numbers to stop multiple identical indexes being constructed whilst retaining order
    row_indexes = list(dict.fromkeys(row_indexes))
    indexes = ", ".join([column_letter.format(i=index) for index in row_indexes])
    return indexes
```

this would come with a mapping change like:

```
    "Outcome_Data": {
        "Outcome": "B{i}",
        "UnitofMeasurement": "C{i}",
        "Relevant project(s)": {"Outcome Indicators (excluding footfall)": "D{i}",  # outcome section
                                "Footfall Indicator": "B{i}"},  # footfall outcome section
        "GeographyIndicator": "E{i}",
        "Amount": "F{i}-O{i}",
        "Higher Frequency": "P{i}",
    },
```